homepage/templates/homepage/reviews/reviews.html

```diff
@@ -1,18 +1,77 @@
+{% extends 'homepage/app_layout.html' %}
+{% load static %}
+{% block content %}
 <html>
 <head>
-<title>Class Rater Reviews</title>
+	<title>Class Rater Reviews</title>
+      <link rel="stylesheet" type="text/css" href="{% static 'css/reviews.css' %}">
 </head>
-
 <body>
-{% extends 'homepage/app_layout.html' %}
-{% block content %}
-<div class="container" style="margin-top:30px">
-  <div class="row">
-    <div class="col-sm-4">
       <h2>Reviews Page</h2>
+      <div class="num-review-title">{{reviews.count}} reviews available</div>
+      <div class="page-container" style="margin-top:30px">
+        <div class="reviews-container">
+            {% for review in reviews %}
+            <div class="review-box">
+             <div class="title-row">
+              <h4>{{review.user}}</h4>
+              <h6>{{review.date}}</h6>
+        </div>
+        <div>
+              About Course: {{review.course.name}}
+        </div>
+        <div style="margin-top: 20px">
+              {% if review.rate == 5 %}
+              {% for i in "12345"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% elif review.rate == 4 %}
+              {% for i in "1234"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% elif review.rate == 3 %}
+              {% for i in "123"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% elif review.rate == 2 %}
+              {% for i in "12"|make_list %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endfor %}
+              {% else %}
+              <img src="{% static 'icons/star.png' %}">
+              {% endif %}
+        </div>
+        <p style="margin-top: 20px">
+              {{review.content}}
+              <div class="extra-details">
+               {% if review.professor %}

```

Comment:
I've noticed that when the professor's name is in the review, the **course_load** appears on the right instead of below.
Is it possible to change that?
![image](https://user-images.githubusercontent.com/57869309/116535298-48ad5780-a8ec-11eb-9cd2-b3c93b4a8653.png)

Reply:
I think it won't look good with nothing on the left side

Reply:
I like it when it's on the right - maybe put it there permanently regardless of prof?